mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/impl/MeshFactoryImpl.java

```diff
@@ -4,6 +4,7 @@
 import com.gentics.mesh.MeshFactory;
```

Comment:
Enterprise does not use MeshFactory for now, so the impl has been fully moved into OrientDB.